config/__init__.py

```
@@ -0,0 +1,178 @@
+                
+            if key in RESTART_REQUIRED_KEYS:
+                return translations["config_updated_restart"].format(key=key)
+                
+            return translations["config_updated"].format(
+                key=key,
+                old_value=old_value,
+                new_value=sanitized_value
+            )
+        except Exception as e:
+            logging.error(f"Error updating config value: {str(e)}")
+            raise ConfigError(f"Configuration update failed: {str(e)}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)